rs/nns/governance/api/src/ic_nns_governance.pb.v1.rs

```diff
@@ -2381,6 +2483,25 @@ pub mod install_code {
         }
     }
 }
+
+#[derive(candid::CandidType, candid::Deserialize, serde::Serialize, comparable::Comparable)]
+#[allow(clippy::derive_partial_eq_without_eq)]
+#[derive(Clone, PartialEq, ::prost::Message)]
+pub struct NewInstallCode {
```

Comment:
Can we give these more meaningful names?  New compared to what?  I'm not clear how to know when to use one versus the other from the names.  

Are these types temporary (i.e. going to be consolidated back to one set of types in a subsequent PR)?

Reply:
👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍

The word "new" has a very very strong tendency towards becoming obsolete (very quickly).

For example, _Windows NT_ was never going to remain "new technology" forever. By contrast, _Windows 95_ will _always_ will be an accurate name, because the fact that it came out in 1995 is an _immutable_ fact (barring backwards time travel).

Reply:
I did the renaming so PTAL.

> For example, Windows NT was never going to remain "new technology" forever. By contrast, Windows 95 will always will be an accurate name, because the fact that it came out in 1995 is an immutable fact (barring backwards time travel).

Just to clarify, the `New` here didn't refer to the fact that the type is new, but that the data of those types added to the NNS Governance didn't exist before the manage_neuron call. I think your example only applies for the former case. OTOH, if one might think the `New` refers to the type, not the object, then the ambiguity might be enough reason to avoid it.